src/models/spaces/charts/chart-data.ts

```diff
@@ -4,9 +4,15 @@ import { ChartDataSetModel, ChartDataSetModelType, ChartColors } from "./chart-d
 export const ChartDataModel = types
   .model("ChartData", {
     name: types.string,
-    dataSets: types.array(ChartDataSetModel)
+    dataSets: types.array(ChartDataSetModel),
+    labels: types.array(types.string)
   })
   .views(self => ({
```

Comment:
These are the chart level labels as opposed to the labels on the datasets (this is where we store the assayed region for future display in the chart)